Pomodoro.DataAccess/Pomodoro.DataAccess.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="6.0.12" />
```

Comment:
we had a comment about this in chat. In this case - it's okay, I just thought that it being used differently